src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/viewmodel/TakeManagementViewModel.kt

```diff
@@ -268,4 +249,24 @@ class TakeManagementViewModel : ViewModel() {
                     ?: ""}"
         }
     }
+
+    private fun enableButtons() {
+        if(contentList.size != 0) {
+            if (activeContent != null) {
+                when(activeContent.start) {
+                    in (contentList.first().start+1)..(contentList.last().start-1) -> {
+                        hasNext.set(true)
+                        hasPrevious.set(true)
+                    }
+                    contentList.first().start -> {
+                        hasPrevious.set(false)
+                    }
+                    contentList.last().start -> {
+                        hasNext.set(false)
+                    }
```

Comment:
This might miss an edge case or two.  Is `hasNext` set for `contentList.first().start`?

Would it be better to replace the whole `when` with something like this:
```
hasPrevious.set(activeContent.start > contentList.first().start)
hasNext.set(activeContent.start < contentList.last().start)
```

Reply:
i do think that would be a better way to do this. Thanks